napari/utils/colormaps/colormaps.py

```diff
@@ -111,6 +118,39 @@ def _low_discrepancy_image(image, seed=0.5, margin=1 / 256):
     return image_out
 
 
+def color_dict_to_colormap(
+    color_dict, background_label=0, empty_color='black'
+):
+    """
+    Generate a color map based on the given color dictionary
+    Parameters
+    ----------
+    color_dict : dict of label and color string matches
+    background_label : int of background label, default to 0
+    empty_color : str of color used as placeholder when color not present
+
+    Returns
+    -------
+    colormap : Colormap with provided control colors
+    label_color_index : dict of label to color control point within colormap
+    """
+    color_dict[background_label] = 'transparent'
```

Comment:
see comment above but I think these should be inside main labels layer not in here